### PR TITLE
Fix for enabling Universal Analytics on mobile devices.

### PR DESCRIPTION
**Pull request by harnash**
Due to global variable whitelisting on mobile devices wgGaStaging
was not visible. Unified mechanism for detecting production environment.

@macbre: please take a look at this

### CODE REVIEW

**Comment by michalroszka** [comment]
LGTM, but @gabrys might want to take a look, as well?

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderGoogleUA.php [hunk 1]
```diff
@@ -48,6 +48,12 @@ static public function onWikiaSkinTopScripts( &$vars, &$scripts, $skin ){
 			$vars['wgGAUserIdHash'] = md5($user->getId() . $wgGAUserIdSalt);
 		}
 
+		global $wgDevelEnvironment, $wgStagingEnvironment;
```
Comment (nandy-andy) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderGoogleUA.php]: I always prefer having global in one place of function body. But it's my personal opinion :japanese_ogre: I don't think our guidelines cover it - feel free to change it or not :smiley_cat:

**Comment by gabrys** [comment]
LGTM. Agree with @nandy-andy's concern.

File: extensions/wikia/AnalyticsEngine/AnalyticsProviderGoogleUA.php (hunk 1 above)
Comment (harnash) [inline extensions/wikia/AnalyticsEngine/AnalyticsProviderGoogleUA.php]: Good point - my mistake. Fixed in 09f44a5424b7c1eba2f2a634a09deb1f042a4692

**Comment by nandy-andy** [comment]
LGTM :)

**Comment by macbre** [comment]
LGTM as well

**Pull request merged**